Editor/IO/Files/CopyFilesToDirectory.cs

```diff
@@ -0,0 +1,40 @@
+using System.IO;
+using System.Threading.Tasks;
+using UnityEditor;
+using UnityEngine;
+
+namespace UniTools.Build
+{
+    [CreateAssetMenu(
+        fileName = nameof(CopyFilesToDirectory),
+        menuName = MenuPaths.IO + nameof(CopyFilesToDirectory)
+    )]
+    public sealed class CopyFilesToDirectory : BuildStep
+    {
+        [SerializeField] private PathProperty[] m_filePaths = default;
```

Comment:
We can name it m_sourceDirectory

Reply:
This I disagree on. I may call this `m_sourceFiles` though for clarity.